docs/analytics.md

```diff
@@ -0,0 +1,132 @@
+
+<i>Last updated: Feb 25, 2019</i>
+
+<br />
+
+# Firefox Monitor Analytics Plan
+
+This is the Analytics plan for Firefox Monitor. It documents our use of Google Analytics and what we do with the information we collect.
+
+<br>
+
+## Analysis
+**Firefox Monitor uses Google Analytics to collect and organize data. We do this to get a better understanding of what is working, and where we still have work to do.**
+
+
+**Captured data also helps provide answers to the following questions:**
+
+
+**Demographic:**
+
+>From which country does the majority of our traffic originate?
+
+>In which language does the majority of our traffic view the site?
+
+>Which browsers are most commonly used to access Firefox Monitor?
+
+>Which devices are most commonly used to access Firefox Monitor?
+
+<br>
+  
+**User Behavior:**
+
+>Do users scan their email?
+
+>Do users scan multiple email addresses?
+
+>What percentage of users sign up receive alerts from Firefox Monitor?
+
+>Are users more likely to sign up for alerts before or after scanning their email?
+
+
+<br />
+
+## Collection
+
+Events are reported using the [Google Analytics Measurement Protocol](https://developers.google.com/analytics/devguides/collection/protocol/v1/).
+
+We collect data for the following events:
+
+- When the page loads (or reloads).
+`hitType` : pageView
+`page` : location.pathName
+<br>
+
+#### Forms:<br><br> `Email Scan Form` &nbsp; `Scan Another Email Form` &nbsp; `Sign Up Form` &nbsp;
+
+- When a form appears on the page.
+`hitType` : event
+`eventCategory` : Form ID
+`eventAction` : View
+`eventLabel` : Page location ID.
```

Comment:
Looking at the GitHub preview, it looks like all of these are being jammed on the same line:

![create_analytics_md____adventures_in_markdown_by_lesleyjanenorton_ _pull_request__815_ _mozilla_blurts-server](https://user-images.githubusercontent.com/557895/53433586-83807180-39a9-11e9-8658-08843fa79ee7.png)

Not sure if you want to do nested `<ul>` lists:

- When a form appears on the page.
  - `hitType` : event
  - `eventCategory` : Form ID
  - `eventAction` : View
  - `eventLabel` : Page location ID.

Or possibly a `<table>` for the tabular data, but that seems maybe a bit heavy:

- When a form appears on the page.

  | Type | Description |
  |:-----|:------------|
  | `hitType` | event |
  | `eventCategory` | Form ID
  | `eventAction` | View
  | `eventLabel` | Page location ID.

Even just indenting each line by 2-4 spaces seems to help:

- When a form appears on the page.
    `hitType` : event
    `eventCategory` : Form ID
    `eventAction` : View
    `eventLabel` : Page location ID.